### ISSUE TITLE
Might have to tweak how we detect the relic page, etc with HCS Guild Info changes

### ISSUE THREAD

**Issue by gschizas** [gschizas/fallenswordhelper#714]
Original [issue 715](https://code.google.com/p/fallenswordhelper/issues/detail?id=715) created by gschizas on 2009-09-23T08:13:58.000Z:

Seems like the new HCS Guild Info changes add in new hidden fields which will 
have to be dealt with.

Relic is known to break, Auction, Quest Book, Advisor Page, Arena Tournament 
and quick scavenging might have issues.

This is a placeholder to look into these potential issues.

basically all the case &quot;-&quot; items in onPageLoad.
